<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inquiry</class>
 <widget class="QWidget" name="Inquiry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="inquiryCal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="result_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>641</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="search_prior">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>all</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>low</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>middle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>high</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>emergent</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="search_type">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>all</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>study</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>play</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>life</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>work</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>优先级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>筛选条件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchName">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="prior_valid">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>该筛选条件是否生效</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="type_valid">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>该筛选条件是否生效</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>在此输入想搜索的任务名（或部分字段）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toSearch">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;按条件查询&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
